sup3r/models/base.py

```diff
@@ -428,6 +430,8 @@ def generate(self, low_res, norm_in=True, un_norm_out=True):
         un_norm_out : bool
            Flag to un-normalize synthetically generated output data to physical
            units
+        exogenous_data : None
+           Placeholder to match multistep model generate function signature
```

Comment:
Is the placeholder necessary? Why not just add a statement doing concatenation to the low res array in this method so that in theory you could provide exo data to a single sup3rmodel?

Reply:
Yeah good call. I think my initial approach made this trick but that's been resolved.